app/src/pages/inside/projectSettingsPageContainer/content/defectTypes/defectTypes.jsx

```diff
@@ -145,7 +145,7 @@ export const DefectTypes = ({ setHeaderTitleNode }) => {
   }, [defectTypes, canAddNewDefectType, isEditable]);
 
   return (
-    <>
+    <div style={{ maxWidth: '1200px', margin: '0 auto' }}>
```

Comment:
Be as kind as to explain the reason to use style instead of className?

Reply:
I would have to create three identical classNames and import them for just 2 properties, there would be a lot more code
(3 lines -> ~20 lines), I thought that this approach was more appropriate here

Reply:
I think that we need to follow recommendations from React official documentation :  https://reactjs.org/docs/dom-elements.html#style.
And if we want to reuse some styles we can use mixins: https://sass-lang.com/documentation/at-rules/mixin .
Also, I saw inline styles in this PR https://github.com/reportportal/service-ui/pull/3363/files.
U can create a mixin and reuse it.

Reply:
Please see the comment below.
And regarding inline style. In this case we can avoid it by creating a reusable component.
Mixins as an option here, but the new component or `projectSettingsPageContainer` styling is more preferable as it is not just styles itself, it is a part of layout. That means it should repeat the nodes structure together with styling.

Generally we try to avoid using inline styles in most cases, because it complicates styles overriding and such styles does not injected from the js to the `*.css` during build process. Another concern of using inline styles here is that it is not reusable, in case of requirements will change, we will need to update all setting pages instead of updating styles just in one place.